dist/utils.js

```diff
@@ -261,5 +261,5 @@ var unwrap = exports.unwrap = function unwrap(maybeArray) {
 
 /** Parses HTML and returns body element */
 var parseHTML = exports.parseHTML = function parseHTML(innerHTML) {
-  return new DOMParser().parseFromString(innerHTML, "text/html").body;
+  return new DOMParser().parseFromString(innerHTML, "application/xhtml+xml");
```

Comment:
Als dit de oplossing is, dan denk ik dat we een boolean property xhtml (default: false) moeten maken. Want in principe is het idee dat je er gewoon html ingooit en dat het werkt. Het is niet de bedoeling dat hij ervan uitgaat dat het xhtml is.